Scalar.Common/ProcessHelper.cs

```diff
@@ -24,6 +24,11 @@ public static ProcessResult Run(string programName, string args, bool redirectOu
             return Run(processInfo);
         }
 
+        public static string GetBundledExternalBinariesLocation()
```

Comment:
The reason we need to place `nuget.exe` inside a subdirectory from the other Scalar binaries is that otherwise `nuget.exe` will try and load and bind with the surrounding framework assemblies, which are .NET Core, leading to a failure to run the .NET Framework targeting `nuget.exe` file.

Placing it inside a subdirectory (away from the .NET Core Scalar binaries) means it can run with a clean 'search path', and therefore _actually run!_.

Reply:
Should "extbin" be a constant somewhere? Do we use it anywhere else?

Reply:
Updated to come from the generated `ScalarConstants`.